src/eth/execute.yaml

```diff
@@ -13,6 +13,12 @@
     name: Return data
     schema:
       $ref: '#/components/schemas/bytes'
+  errors:
+    - code: 3
```

Comment:
I suggest to have an error code similar to what we have on the engine AP : https://github.com/ethereum/execution-apis/blob/4140e528360fea53c34a766d86a000c6c039100e/src/engine/common.md?plain=1#L91
Choose a close number like -31000

Reply:
I'm using code 3 here because it is the de-facto error code for this condition. There is no requirement for codes to be negative - the [JSON-RPC spec says](https://www.jsonrpc.org/specification#error_object) "The error codes from and including -32768 to -32000 are reserved for pre-defined errors. ... The remainder of the space is available for application defined errors."